src/js/monkeys/desktop.js

```diff
@@ -12,26 +12,57 @@ desktop.calculateSize = function () {
 };
 
 desktop.generateHtml = function (data) {
-  var $monkey = $('<div />').addClass('Heidelberg-Book with-Spreads desktop');
+  var $monkeyWrapper = $('<div />')
+    .addClass('pos-relative monkey-wrapper js--add-overlay');
+  var $monkey = $('<div />').addClass('Heidelberg-Book with-Spreads desktop pos-absolute');
 
   $.each(data.urls, function (i, url) {
     $('<div />')
-      .addClass('Heidelberg-Spread page-' + data.letters[i].type)
+      .addClass('Heidelberg-Spread page-' + data.letters[i].type + ' Page-' + i)
       .append($('<img />').attr('src', url))
       .appendTo($monkey);
   });
+  $monkey.appendTo($monkeyWrapper);
 
-  return $monkey;
+  return $monkeyWrapper;
 };
 
 desktop.init = function (data, $events) {
   var maxBookProgress = 0;
 
-  data.heidelberg = new Heidelberg(data.html, {
+
+  Heidelberg.prototype.replacePage = function(pages, spreads) {
+    console.log("replaciong page");
+    var page = $('.Page' + pages);
+  }
+
+  data.heidelberg = new Heidelberg(data.html.find('.Heidelberg-Book'), {
     arrowKeys: false,
     hasSpreads: true
   });
 
+
+  data.swapPage = function(index, character) {
+    var page = (index + 3) + index;
+    var $newImage = $('<img />')
+      .attr('src', character.url1 + data.queryString)
+    var $newImage2 = $('<img />')
+      .attr('src', character.url2 + data.queryString)
```

Comment:
I'm pretty sure I deprecated `data.queryString` (with a comment), might be worth undeprecating it